tests/common/print_utils.rs

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2023-2024 Retake, Inc.
+//
+// This file is part of ParadeDB - Postgres for Search and Analytics
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+use anyhow::Result;
+use prettytable::{format, Cell, Row, Table};
+use std::fmt::{Debug, Display};
+
+pub trait Printable: Debug {
+    fn to_row(&self) -> Vec<String>;
+}
+
+macro_rules! impl_printable_for_tuple {
+    ($($T:ident),+) => {
+        impl<$($T),+> Printable for ($($T,)+)
+        where
+            $($T: Debug + Display,)+
+        {
+            #[allow(non_snake_case)]
+            fn to_row(&self) -> Vec<String> {
+                let ($($T,)+) = self;
+                vec![$($T.to_string(),)+]
+            }
+        }
+    }
+}
+
+// Implement Printable for tuples up to 12 elements
+impl_printable_for_tuple!(T1);
```

Comment:
Let's not bundle this into this PR, it's a separate thing that we can discuss whether it's worth bringing into the repo.

Reply:
Sure, done. I've removed the macros. Let me know if you want `tests/common/print_utils.rs` removed as well.